Bottom: cbd81b2790ffe2326d9326c63e5a272c3f172ba1
Top:    4ba810928c534b97666dff271e7fb3c3e7fa98c4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-27 16:08:21 +0100

Refresh of create-a-private-structure-for

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index f783ecc..9641a25 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -2677,6 +2677,8 @@ ghid_parse_arguments (int *argc, char ***argv)
   gport->zoom = 300.0;
   pixel_slop = 300;
 
+  ghid_init_renderer (argc, argv, gport);
+
   ghid_config_files_read (argc, argv);
 
   Settings.AutoPlace = 0;
